<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LdapAuthenticator</class>
 <widget class="QMainWindow" name="LdapAuthenticator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>360</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>768</width>
    <height>288</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LDAP Authenticator</string>
  </property>
  <property name="styleSheet">
   <string>
/* Main window background: Dark Gray */
QMainWindow {
    background: #555555;              /* deep dark gray background */
    font-family: &quot;Segoe UI&quot;, Roboto, Arial, sans-serif;
    font-size: 14px;
    color: #E0E0E0;                   /* very light gray text */
}

/* Input Fields */
QLineEdit {
    background: #777777;              /* darker input background */
    border: 2px solid #444444;
    border-radius: 6px;
    padding: 6px;
    color: #E0E0E0;
}
QLineEdit:focus {
    border-color: #5DADE2;
}

/* Buttons */
QPushButton {
    background-color: #3B82F6;        /* strong blue button */
    color: #FFFFFF;
    border-radius: 6px;
    padding: 8px 16px;
}
QPushButton:hover {
    background-color: #60A5FA;        /* lighter on hover */
}
QPushButton:pressed {
    background-color: #2563EB;        /* deeper blue when pressed */
}

/* Tree Widget */
QTreeWidget {
    background-color: #252525;        /* slightly lighter than main window */
    border: 1px solid #444444;
    border-radius: 4px;
    color: #E0E0E0;
}
QHeaderView::section {
    background-color: #3B82F6;
    color: #FFFFFF;
    padding: 4px;
    font-weight: bold;
}
QTreeWidget::item:nth-of-type(odd) {
    background: #2F2F2F;
}
QTreeWidget::item:selected {
    background-color: #60A5FA;
    color: #FFFFFF;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLineEdit" name="lineEditHost">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButtonConnect">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="pushButtonLogin">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="lineEditDN">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="pushButtonEye">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="labelPassword">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLineEdit" name="lineEditPassword">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::EchoMode::Password</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelDN">
      <property name="font">
       <font>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Distinguished Name</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="labelStatus">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="labelHost">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Host:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" rowspan="8">
     <widget class="QTreeWidget" name="treeWidgetAttributes">
      <column>
       <property name="text">
        <string>Attribute</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Value</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
